Makefile

```diff
@@ -20,8 +20,9 @@ image:
 bufbuild-image:
 	docker build -f bufbuild.Dockerfile . -t caraml-store-bufbuild:build
 
+# LegacyJobService is required on server side for backward compatibility with old clients but not required for the new SDK
 bufbuild-proto:
-	docker run --volume "${PWD}:/caraml-store" --workdir /caraml-store/caraml-store-protobuf/src/main/proto caraml-store-bufbuild:build generate --template buf.gen.sdk.yaml
+	docker run --volume "${PWD}:/caraml-store" --workdir /caraml-store/caraml-store-protobuf/src/main/proto caraml-store-bufbuild:build generate --template buf.gen.sdk.yaml --exclude-path feast/core/LegacyJobService.proto
```

Comment:
why the proto is not deleted? is this still in use by feast ingestion?

Reply:
It is required for the server side to support older client. bufbuild-proto is only use to generate the new client.